modules/op-creating-pipeline-templates-admin-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/working-with-pipelines-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="op-creating-pipeline-templates-admin-console_{context}"]
+= Creating pipeline templates in the Administrator perspective
+
+As a cluster administrator, you can create pipeline templates which developers can reuse when they create a pipeline on the cluster.
```

Comment:
Please double-check the use of "which" re IBM SG. In the PDF copy that I have, "that" and "which" are present, and I don't think this particular usage works according to them. I am not 100% sure, however.